docs/interpreter/jdbc.md

```diff
@@ -165,6 +165,10 @@ There are more JDBC interpreter properties you can specify like below.
     <td>The path to the keytab file</td>
   </tr>
   <tr>
+      <td>zeppelin.jdbc.auth.kerberos.proxy.enable</td>
+      <td>true or false. when auth type is kerberos, whether do the kerberos proxy behavior with the login user (if not anonymous) to get the connection, default value is true</td>
```

Comment:
yeah, I think you are right,  your description is more understandable, thx for your advice，It has been updated